### ISSUE TITLE
Disable progressbar?

### ISSUE THREAD

**Issue by mayur-who** [fizyr/keras-retinanet#1460]
Is there a way to Disable progressbar during the training in the Evaluate callback? It messes up the whole output and it becomes difficult to babysit the training process.

We can add a flag in Evaluate callback and can use progressbar.NullBar instead

**Comment by mayur-who** [comment]
Forgot to attach the console output:


`677/677 [==============================] - 574s 848ms/step - loss: 1.9491 - regression_loss: 1.6675 - classification_loss: 0.2815 - val_loss: 2.1217 - val_regression_loss: 1.8144 - val_classification_loss: 0.3073
Epoch 7/20
677/677 [==============================] - ETA: 0s - loss: 1.9165 - regression_loss: 1.6399 - classification_loss: 0.2766Running network: 100% (601 of 601) |#####| Elapsed Time: 0:00:45 Time:  0:00:45
Parsing annotations: 100% (601 of 601) |#| Elapsed Time: 0:00:00 Time:  0:00:00
958 instances of class pneumonia with average precision: 0.5143`

**Comment by mayur-who** [comment]
stale bot, shut up.

**Comment by hgaiser** [comment]
There's currently no way to do this. I think it is caused by the progressbar not fitting on one line, but I'm not sure. You could try to run it in a wider terminal to see if that solves your problem. Alternatively, a flag to change the verbosity of the progress would be required.

**Comment by mayur-who** [comment]
The resolution/terminal column width is not an issue here. The progress bar output along with model training output messes this up. 
yeah I agree, I know there is no way to do this without modifying the code and having an option to disable the progress bar. 
Since the repo is not under active development, we can close this issue.
